errbot/bootstrap.py

```diff
@@ -74,6 +75,12 @@ def bot_config_defaults(config):
 
 
 def setup_bot(backend_name: str, logger, config, restore=None) -> ErrBot:
+    # Check if git is installed on the system and in the path. If not, error out immediately
+    if which("git") is None:
+        log.exception("git is either not installed or not accessible in the path. Please install it or add it to your "
```

Comment:
I think something like this may read better. `"The required 'git' package is either not installed or accessible in your system path. Please install 'git' or ensure it's added to your system path."`